### PR TITLE
Update login test

### PR DESCRIPTION
**Pull request by kajalrahi**
Added test for Delete Account.

### CODE REVIEW

File: page-object/home/HomePage.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { Locator, Page } from "@playwright/test";
+import BasePage from "./BasePage";
+
+const { expect } = require('@playwright/test')
+
+class Homepage extends BasePage{
```
Comment (Rajeevrahi5) [inline page-object/home/HomePage.ts:6]: Add space before {

File: page-object/onboarding/DeletePage.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Locator, Page } from "@playwright/test";
+import BasePage from "../home/BasePage";
+const { expect } = require('@playwright/test');
+
+class DeletePage{
```
Comment (Rajeevrahi5) [inline page-object/onboarding/DeletePage.ts]: Add space before {

File: page-object/onboarding/DeletePage.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    constructor(page: Page) {
+        this.page = page
+        this.linkContinue = page.getByRole('link', { name: 'Continue' })
+        this.textLoggedUsernameVisible = page.getByRole('link', { name: ' Logout' })
+        this.buttonDelete = page.getByRole('link', { name: ' Delete Account' })
+        this.textVerifyAcoountDeleted = page.getByText('Account Deleted!')
+        this.buttonContinue = page.getByRole('link', { name: 'Continue' })
+    }
+
+    async deleteAccount(){
```
Comment (Rajeevrahi5) [inline page-object/onboarding/DeletePage.ts]: Add space before {

File: page-object/onboarding/SignupPage.ts [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+      this.dropdownSelectCountry = page.getByLabel('Country *')
+      this.textState = page.getByLabel('State *')
+      this.textCity = page.getByLabel('City *' )
+      this.textZipcode = page.locator('#zipcode')
+      this.textMobileNumber = page.getByLabel('Mobile Number *')
+      this.buttonCreateAccount = page.getByRole('button', { name: 'Create Account' })
+      this.popupAccountCreatedMessage = page.getByText('Account Created!')
+    }
+
+    async signupAutomationExercise(name: string, emailId: string){
```
Comment (Rajeevrahi5) [inline page-object/onboarding/SignupPage.ts]: Add space before {

File: page-object/onboarding/SignupPage.ts [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+      this.chooseTitle(title)
+      await expect (this.textNameVerify).toBeVisible()
+      await expect(this.textEmailidVerify).toBeVisible()
+      await this.textPassword.fill(password)
+      this.selectDateOfBirth(day, month,year)
+      await this.buttonSignupForNewsletter.click()
+      await this.buttonRecieveSpecialOffers.isVisible()  
+    }
+
+    async selectDateOfBirth(day:string, month:string, year:string){
```
Comment (Rajeevrahi5) [inline page-object/onboarding/SignupPage.ts]: Add space before {

File: page-object/onboarding/SignupPage.ts [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+    async chooseTitle(title: string) {
+        if(title === 'Mr.') {
+           await this.radioButtonMr.click()
+        } else {
+            await this.radioButtonMrs.click()
+        }
+      }
+
+    async fillAdressInformation(firstName:string, lastName:string, company:string, address:string, address2:string, country:string, state:string, city:string, zipcode:string, mobileNumber:string){
```
Comment (Rajeevrahi5) [inline page-object/onboarding/SignupPage.ts]: Add space before {

File: tests/onboarding/DeleteTest.spec.ts [hunk 7]
```diff
@@ -0,0 +1,43 @@
+import {test, expect} from '@playwright/test'
+import Homepage from '../../page-object/home/HomePage'
+import DeletePage from'../../page-object/onboarding/DeletePage'
+import SignupPage from '../../page-object/onboarding/SignupPage'
+import UtilityHelper from '../../utility/UtilityHelper'
+
+const testData = JSON.parse(JSON.stringify(require('../../test-data/test-data.json')))
+
+test('To delete account',async({page})=> {
```
Comment (Rajeevrahi5) [inline tests/onboarding/DeleteTest.spec.ts]: Description of the test should be `Test Delete Account` and there should be a space before =>

File: tests/onboarding/SignupTest.spec.ts [hunk 8]
```diff
@@ -0,0 +1,39 @@
+import {test, expect} from '@playwright/test'
+import SignupPage from '../../page-object/onboarding/SignupPage'
+import Homepage from '../../page-object/home/HomePage'
+import UtilityHelper from '../../utility/UtilityHelper'
+import DeletePage from '../../page-object/onboarding/DeletePage'
+const testData = JSON.parse(JSON.stringify(require('../../test-data/test-data.json')))
+
+
+test('Signup',async({page})=> {
```
Comment (Rajeevrahi5) [inline tests/onboarding/SignupTest.spec.ts]: Description should be `Register a new user` and add space before =>

File: utility/UtilityHelper.ts [hunk 9]
```diff
@@ -0,0 +1,6 @@
+class UtilityHelper{
```
Comment (Rajeevrahi5) [inline utility/UtilityHelper.ts]: Add space before {

File: utility/UtilityHelper.ts [hunk 10]
```diff
@@ -0,0 +1,6 @@
...
+
+    async randomEmailGenerator(){
```
Comment (Rajeevrahi5) [inline utility/UtilityHelper.ts]: Add space before {

**Review by Rajeevrahi5 — CHANGES_REQUESTED** [review]
Add the review comment in you branch and push

File: page-object/home/HomePage.ts (hunk 1 above)
Comment (kajalrahi) [reply page-object/home/HomePage.ts:6]: fixed

File: page-object/onboarding/DeletePage.ts (hunk 3 above)
Comment (kajalrahi) [reply page-object/onboarding/DeletePage.ts]: fixed

File: page-object/onboarding/SignupPage.ts (hunk 5 above)
Comment (kajalrahi) [reply page-object/onboarding/SignupPage.ts]: fixed

File: tests/onboarding/DeleteTest.spec.ts (hunk 7 above)
Comment (kajalrahi) [reply tests/onboarding/DeleteTest.spec.ts]: fixed

File: tests/onboarding/SignupTest.spec.ts (hunk 8 above)
Comment (kajalrahi) [reply tests/onboarding/SignupTest.spec.ts]: fixed

File: utility/UtilityHelper.ts (hunk 10 above)
Comment (kajalrahi) [reply utility/UtilityHelper.ts]: fixed

File: page-object/onboarding/SignupPage.ts (hunk 4 above)
Comment (kajalrahi) [reply page-object/onboarding/SignupPage.ts]: fixed

File: utility/UtilityHelper.ts (hunk 9 above)
Comment (kajalrahi) [reply utility/UtilityHelper.ts]: fixed

**Pull request merged**